### PR TITLE
feat: patch oclif node execution so there are no warnings

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

**Review by nahsi — APPROVED** [review]

**Comment by akim-bow** [comment]
Maybe it's better to set env variable in package.json run script or somewhere else? This solution strongly depends on oclif source code and for me looks brittle.

File: src/beforeBuild.ts [hunk 1]
```diff
@@ -97,3 +97,30 @@ await mkdir(COMPILED_INSTALLATION_SPELL_AQUA_WITH_TRACING_PATH, {
...
 await compileInstallationSpellAqua();
 await compileInstallationSpellAqua(true);
+
+const BIN_FILE_PATH = join(
+  join("node_modules", "oclif", "lib", "tarballs"),
+  "bin.js",
+);
+
+const binFileContent = await readFile(BIN_FILE_PATH, FS_OPTIONS);
+const NODE_RUN = `  "\\$NODE" "\\$DIR/run" "\\$@"`;
```
Comment (akim-bow) [inline src/beforeBuild.ts:107]: What if oclif will change this line in the next version?

Comment (shamsartem) [reply src/beforeBuild.ts:107]: Most likely the error will be thrown, as you can see from the code further down below

**Comment by shamsartem** [comment]
> Maybe it's better to set env variable in package.json run script or somewhere else? This solution strongly depends on oclif source code and for me looks brittle.

there is no "run script in package.json". If you read this carefully you will see that this is a shell script which directly uses Node.js binary for executing js file. Because these shell scripts are a part of oclif's source code, which dictates the packaging process and the update mechanism (as I showed on the call) - I don't see a cleaner way to do this, then modifying it

**Pull request merged**